enterprise/cypher/physical-planning/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/SlotAllocationTest.scala

```diff
@@ -28,6 +28,7 @@ import org.neo4j.cypher.internal.ir.v3_3.{CardinalityEstimation, IdName, Planner
 import org.neo4j.cypher.internal.v3_3.logical.plans.{Ascending, _}
 import org.neo4j.cypher.internal.v3_3.logical.{plans => logicalPlans}
 
+//noinspection NameBooleanParameters
```

Comment:
Can't we name the boolean parameters instead?

Reply:
We can, but I think it makes for less readable code in this case.

Reply:
ok, you win